package.json

```diff
@@ -116,7 +116,8 @@
     "xterm-addon-webgl": "0.11.1",
     "yauzl": "^2.9.2",
     "yazl": "^2.4.3",
-    "zone.js": "^0.8.4"
+    "zone.js": "^0.8.4",
+    "azdataGraph": "git://github.com/kburtram/azdataGraph.git#0.0.4"
```

Comment:
Will you be putting this in the MS org at some point? All code we own should generally go there.

Reply:
we'll be moving to microsoft org in follow-up PR prior to Feb release.